.github/workflows/docs.yml

```diff
@@ -154,15 +150,41 @@ jobs:
         uses: nwtgck/actions-netlify@v3.0
         with:
           publish-dir: .
-          production-deploy: ${{ github.ref == 'refs/heads/main' }}
-          github-token: ${{ secrets.GITHUB_TOKEN }}
+          production-deploy: ${{ github.event.pull_request.head.repo.full_name == github.repository && github.ref == 'refs/heads/main' }}
```

Comment:
> What is this part intended to do? github.event.pull_request.head.repo.full_name == github.repository

This conditions ensures we are in `pyvista/pyvista` (only true for this repository) and will be false for forks like `banesullivan/pyvista`

>I also cannot find a reference for the head key in a pull_request object

Hm. I copied this from

https://github.com/pyvista/pyvista/blob/a98885564ae6bbb8a58f98c5ca27d3b97b142cc4/.github/workflows/docker-package.yml#L26

But looking at the docs it may be `headRef`

Reply:
This check really isn't needed as `github.ref == 'refs/heads/main'` should be sufficient here

Reply:
removed this whole check and limited it to just `github.ref == 'refs/heads/main'`